js/modules/timeline.src.js

```diff
@@ -248,6 +283,7 @@ seriesType('timeline', 'line', {
         if (points) {
             xMap = series.mappedXData = this.getMappedXData();
 
+            // Apply new calculated point values.
             each(points, function (point, i) {
```

Comment:
Consider to use series.points and do not declare points variable which is used only once.